integration/tests/dialog-time-picker.test.js

```diff
@@ -0,0 +1,71 @@
+import { expect } from '@esm-bundle/chai';
+import { fixtureSync, nextFrame, nextRender } from '@vaadin/testing-helpers';
+import { sendKeys } from '@web/test-runner-commands';
+import './not-animated-styles.js';
+import '@vaadin/dialog';
+import '@vaadin/time-picker';
+
+describe('time-picker in dialog', () => {
+  let dialog, timePicker;
+
+  beforeEach(async () => {
+    dialog = fixtureSync('<vaadin-dialog></vaadin-dialog>');
+    dialog.renderer = (root) => {
+      root.innerHTML = '<vaadin-time-picker></vaadin-time-picker>';
+    };
+    dialog.opened = true;
+    await nextFrame();
+    timePicker = dialog.$.overlay.querySelector('vaadin-time-picker');
+  });
+
+  afterEach(() => {
+    dialog.opened = false;
```

Comment:
**nit:** The dialog closes itself in `disconnectedCallback` so manual closing seems to be unnecessary.

Reply:
Thanks, removed.